thirdeye-server/src/main/java/ai/startree/thirdeye/auth/oauth/OAuthConfiguration.java

```diff
@@ -19,13 +19,23 @@
 import java.util.Map;
 
 public class OAuthConfiguration {
+  private boolean enabled;
   private String serverUrl;
   private String keysUrl;
   private String clientId;
   private List<String> required = new ArrayList<>();
   private Map<String, Object> exactMatch = new HashMap<>();
   private OauthCacheConfiguration cache;
 
+  public boolean isEnabled() {
```

Comment:
current behavior of this is `true` so setting it to false by default (by not defining it) can cause problems. recommend, we take this up separately.

Reply:
Valid point. Reverted the introduction of this flag